plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,10 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note: Katello 3.15 changes the default database for pulp from postgres v9.x to postgres v12. External databases should be upgraded to avoid conflicts.
```

Comment:
@ianballou I would like to establish necessary messaging here, and propagate those backwards into each of the extant version. That way, any edits or changes we'd only have to change here, and then when final, add those messages to the other versions.